bin/osg-notify

```diff
@@ -115,6 +115,9 @@ def parseargs():
                          "(default: all)", )
     oparser.add_argument("--bypass-dns-check", action="store_true", dest="bypass_dns_check",
                          help="Bypass checking that one of the host's IP addresses matches with the hostanme resolution")
+    oparser.add_argument("--force-non-ascii", action="store_true", dest="force_non_ascii",
```

Comment:
How about calling the option `--allow-non-ascii`?  (rather than `force`)

Reply:
done